app/assets/javascripts/application.js

```diff
@@ -1,16 +1,15 @@
-// This is a manifest file that'll be compiled into application.js, which will include all the files
-// listed below.
-//
-// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
-// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
-//
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
-// compiled file. JavaScript code in this file should be added after the last require_* statement.
-//
-// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
-// about supported directives.
-//
 //= require jquery
+//= require bootstrap-sprockets
 //= require jquery_ujs
-//= require turbolinks
 //= require_tree .
+
+$(document).ready(function(){
+  $(function(){
+    $('#import-form').submit(function (e) {
+      if ($(this).find('input[name="file"]').val() == '') { 
+        e.preventDefault();
+        alert('Select a file first');
+      }     
+    });
```

Comment:
Assuming that this is the first part of a more complex program, how would you make it better, in the sense of not colliding with forms in other pages?